packages/npm-dependencies-validation/src/utils/packageJsonUtil.ts

```diff
@@ -17,17 +18,18 @@ const pnpmManagerFiles = [
 ];
 const monorepoProps = ["workspaces"];
 
-async function readJsonFile(jsonFilePath: string): Promise<any> {
+async function readJsonFile(jsonFilePath: string): Promise<PackageJson> {
```

Comment:
do we use fs-extra here?
I think it has a readJson utility method

Reply:
no I do not use fs-extra, I think it is redundant for this package